compiler/stable_mir/src/mir/body.rs

```diff
@@ -246,7 +246,6 @@ pub enum AssertMessage {
     RemainderByZero(Operand),
     ResumedAfterReturn(CoroutineKind),
     ResumedAfterPanic(CoroutineKind),
-    MisalignedPointerDereference { required: Operand, found: Operand },
```

Comment:
Can you please just mark this as deprecated for now instead or removing it? Thanks